public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -176,7 +178,14 @@ export class CloudWatchDatasource
     logQueries: CloudWatchLogsQuery[],
     options: DataQueryRequest<CloudWatchQuery>
   ): Observable<DataQueryResponse> => {
-    const validLogQueries = logQueries.filter((item) => item.logGroupNames?.length);
+    const queryParams = logQueries.map((target: CloudWatchLogsQuery) => ({
+      queryString: target.expression || '',
+      refId: target.refId,
+      logGroupNames: this.getActualLogGroups(target.logGroupNames),
```

Comment:
nit: I think this is more readable without the function:

```suggestion
      logGroupNames: target.logGroupNames || this.defaultLogGroups;,
```